sormas-backend/src/main/java/de/symeda/sormas/backend/user/User.java

```diff
@@ -315,4 +316,9 @@ public void setLanguage(Language language) {
 	public boolean hasAnyUserRole(UserRole... userRoles) {
 		return Arrays.stream(userRoles).anyMatch(getUserRoles()::contains);
 	}
+
+	@Transient
+	public JurisdictionLevel getJurisdictionLevel(){
```

Comment:
Pretty sure you didn't use code formatting in this class

Reply:
indeed, formatted it, space is there now